src/main/lang/de.json

```diff
@@ -579,6 +579,10 @@
           "weaponSheet": "Waffenbogen"
         }
       },
+      "spell": {
+        "singular": "Beschwörung",
+        "plural": "Beschwörungen"
```

Comment:
I still think "Magieformel" is a better more generalized term, but we can leave it like this for now.